core/storage/suggestion/gae_models_test.py

```diff
@@ -171,6 +171,43 @@ def test_score_type_contains_delimiter(self) -> None:
             self.assertTrue(
                 suggestion_models.SCORE_CATEGORY_DELIMITER not in score_type)
 
+    def test_get_translation_suggestions_submitted_before_given_date(
```

Comment:
Update test names to reflect range as well. Ditto below.

Reply:
Done